simbad/command_line/simbad_database.py

```diff
@@ -143,7 +145,15 @@ def create_lattice_db(database):
             'unitCellAngleGamma,spaceGroup,experimentalTechnique&service=wsfile&format=csv'
 
     crystal_data, error_count = [], 0
-    for line in urllib2.urlopen(url):
+
+    rcsbReportFile=os.path.join(os.environ["CCP4_SCR"], "rcsb" + "_%s.csv" % str(datetime.datetime.now()).replace(" ","_").replace("-","_").replace(":","_").replace(".","_")) 
```

Comment:
Should probably keep this to 80 lines and use snake case to conform to PEP8 guidelines.

Reply:
no need for the 80 line characters but certainly spaces between assignment operator and snake_case. also, use new `str.format` instead of `%`.